ssn/integrated/ssn.ttl

```diff
@@ -59,9 +59,21 @@ sosa:FeatureOfInterest
 ssn:Property a owl:Class ;
```

Comment:
This class is now deprecated. Should the axiomatization still be updated?

Reply:
Clarified in new commit 351438d